test/integration/helpers.py

```diff
@@ -79,12 +79,14 @@ def wait_for_condition(
 
 
 # Retry function to help in case of requests sending too quickly before instance is ready
-def retry_sending_request(retries: int, condition: Callable, *args) -> object:
+def retry_sending_request(
+    retries: int, condition: Callable, *args, **kwargs
```

Comment:
Added **kwargs here so I could specify `disk_encryption` in the rebuild arguments 👍